sky/task.py

```diff
@@ -225,8 +225,8 @@ def __init__(
 
         self.inputs = None
         self.outputs = None
-        self.estimated_inputs_size_gigabytes = None
-        self.estimated_outputs_size_gigabytes = None
+        self.estimated_inputs_size_gigabytes = 0.0
+        self.estimated_outputs_size_gigabytes = 0.0
```

Comment:
Why changing these values?

Reply:
It is because we use the value in the optimizer, but not correctly handle `None`. I changed the handling in the Optimizer instead. Thanks for catching this.